ui/ui.go

```diff
@@ -151,9 +151,13 @@ func newModel(cfg Config) tea.Model {
 		stash:  newStashModel(&common),
 	}
 
-	info, err := os.Stat(cfg.Path)
+	path := cfg.Path
+	if path == "" {
+		path = "."
+	}
+	info, err := os.Stat(path)
```

Comment:
This is an unrelated bug fix, that I believe was introduced by me on my previous PR. When no argument is given, we should default to the current directory.